assets/openobserve/ingest.py

```
@@ -0,0 +1,53 @@
+
+user = "root@clpbench.com"
+password = "password"
+bas64encoded_creds = base64.b64encode(bytes(user + ":" + password, "utf-8")).decode("utf-8")
+
+headers = {"Content-type": "application/json", "Authorization": "Basic " + bas64encoded_creds}
+org = "default"
+stream = "clpbench1"
+openobserve_host = "http://localhost:5080"
+openobserve_url = openobserve_host + "/api/" + org + "/" + stream + "/_json"
+
+def base32_encode_string(s):
```

Ruff (0.12.2)

Missing return type annotation for public function `base32_encode_string`

(ANN201)